### PR TITLE
Styling tweaks

### PR DESCRIPTION
**Pull request by zarina-m**
Some styling tweaks:

- Display the correct rating for each restaurant as stars
- Display the correct price for each restaurant
- Padding on a few elements

image

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -106,9 +168,7 @@ restaurantsApp.handleSearch = function () {
     let locationInput = $('#locationInput').val().trim(' ');
     let termInput = $('#termInput').val().trim(' ');
     //  Get user city and populate the locationTerm input
```
Comment (Mila-ashtari) [inline script.js]: could you delete this cm , I think I left it here

Comment (zarina-m) [reply script.js]: Sure, removed.

File: script.js [hunk 2]
```diff
@@ -51,6 +51,48 @@ restaurantsApp.getReviews=function(result){
...
+                <h2>${businessName}</h2>
+                <span>${ratingString}</span>
+                <span>${priceString}</span> 
+                <h3>${businessAddress}</h3>
+                </div>
+                </div>`
+        //Displays each result to the page
+        $('.listOne').append(html)
+    }
+
```
Comment (Mila-ashtari) [inline script.js]: this seems to be left from before, there is an updated displayDetails function further down

**Review by Mila-ashtari — COMMENTED** [review]
See my cms please

File: script.js (hunk 2 above)
Comment (zarina-m) [reply script.js]: good catch!

**Pull request merged**